src/lib/yang/schema.lua

```diff
@@ -700,10 +707,11 @@ function resolve(schema, features)
                   node.body[k] = v
                end
                -- TODO: Handle refine and augment statements.
-            else
-               node.body[k] = visit(v, env)
             end
          end
+         for k,v in pairs(node.body or {}) do
```

Comment:
So. Instead of doing this here:

```lua
for k,v in pairs(node.body or {}) do
   node.body[k] = visit(v, env)
end
```

You want something similar that runs on the result of `resolve` that handles inheriting the config. I think you'd still want to do this iteration though as there might be other things to inline or otherwise handle within the nodes (for example inlining typedefs or other groupings, etc.).

I don't understand how moving config inheritance out of here helps anything?